server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
+            if (userEnteredSecret != null && !secret.equals(userEnteredSecret)) {
+              log.info("Aborted reset security: Instance secret did not match what is in accumulo-site.xml");
+              return;
+            }
+            log.info("Beginning user initiated reset security on accumulo.");
```

Comment:
I added this log message since it is not clear what is going on in the output...
2016-09-14 12:35:03,251 [conf.AccumuloConfiguration] INFO : Loaded class : org.apache.accumulo.server.security.handler.ZKAuthorizor
2016-09-14 12:35:03,252 [conf.AccumuloConfiguration] INFO : Loaded class : org.apache.accumulo.server.security.handler.ZKAuthenticator
2016-09-14 12:35:03,253 [conf.AccumuloConfiguration] INFO : Loaded class : org.apache.accumulo.server.security.handler.ZKPermHandler
2016-09-14 12:35:03,508 [handler.ZKAuthenticator] INFO : Removed /accumulo/5a96e1b5-452b-4c35-ada7-cfa807cbd994/users/ from zookeeper